core_tester/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
```

Comment:
Let's just do what Deno does: rename the directory to `tests`, and make it a lib in `Cargo.toml`:

https://github.com/denoland/deno/tree/main/tests

Reply:
done!